### PR TITLE
platypusvar 0.8.1 (new formula): create

### PR DESCRIPTION
**Pull request by tmozgach**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [ ] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ ] Not altered the `bottle` section?
- [ ] Checked that the tests still pass?

### CODE REVIEW

**Comment by tmozgach** [comment]
@sjackman Could you please check and merge a new formula?

File: platypusvar.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Platypusvar < Formula
+  desc "It is variant-detection in high-throughput sequencing data"
+  homepage "http://www.well.ox.ac.uk/platypus"
+  url "http://www.well.ox.ac.uk/bioinformatics/Software/Platypus-latest.tgz"
+  version "0.8.1"
+  sha256 "a0f39e800ebdc5590e9b568a791bc6746df0fde4d1c3622140db64dea175622b"
+
+  depends_on "htslib"
+  depends_on "python"
```
Comment (sjackman) [inline platypusvar.rb]: Please change this line to
```ruby
depends_on :python unless OS.mac?
```

**Comment by tmozgach** [comment]
@sjackman done!

**Comment by sjackman** [comment]
Merged. Thanks, Tanya!

For future commits the commit log message of a new formula should be
`platypusvar 0.8.1 (new formula)`
rather than
`platypusvar 0.8.1 (new formula): create`

**Pull request closed without merging**